packages/svelte/src/styles/index.css

```
@@ -4,3 +4,52 @@
 @theme {
 	--breakpoint-xs: 30rem;
 }
+
+@utility container-sizing {
+	&[data-xs='true'] {
```

Stylelint (17.7.0)

[error] Missing scoping root (nesting-selector-no-missing-scoping-root)

(nesting-selector-no-missing-scoping-root)